docs/Server.md

```diff
@@ -418,6 +418,28 @@ Set a default
 Note that this is needed to offer the graceful "close" experience when
 using http2. Node core defaults this to `0`.
 
+<a name="framework-errors"></a>
+### `frameworkErrors`
+
++ Default: `null`
+
+Configure custom error handlers.
```

Comment:
This could use some more information. What is the purpose of configuring such handlers? If I configure a handler, what problem am I solving?

Reply:
What about:

Fastify provides default error handlers for the most common use cases.
Using this option it is possible to override one or more of those handlers with custom code.
`
const fastify = require('fastify')({
  frameworkErrors (error, req, reply) {
    if (err instanceof BadUrlError) {
      const headers = { 
        'Content-Type': 'text/plain'
      } 
      res.writeHead(400, headers) 
      return res.end("Provided url is not valid")
    } else {
      reply.send(err)
    }
  }
})
`

Reply:
That's better.